src/app/modules/admin-shared/confirm-dialog/confirm-dialog.component.html

```diff
@@ -1,28 +1,11 @@
-<!-- <h1 mat-dialog-title style="padding:15px">
-  {{title}}
-</h1>
-
-<div mat-dialog-content>
-  <p>{{message}}</p>
-</div>
-
-<div style="padding-left: 15px;" mat-dialog-actions>
-  <button mat-button (click)="onDismiss()">No</button>
-  <button mat-raised-button color="primary" (click)="onConfirm()">Yes</button>
-</div> -->
-
-
 <mat-toolbar role="toolbar" class="task-header">
-  <!-- <mat-icon style="color: white;">account_circle</mat-icon> -->
-  <h5 class="title">Confirmation</h5>
+  <div class="title">{{data?.title}}</div>
```

Comment:
This component should be in shared module

Reply:
It is in shared module only